### PR TITLE
Word meaning

### PR DESCRIPTION
**Pull request by Sasafrass**
See https://github.com/Sasafrass/straattaal/issues/30.

Current status:
* Implement and display Meaning Form.
* Update Generate Slang Form + functionality in /main/routes.py to prevent bug where it would always submit the generate slang form, even when just submitting the meaning form.
* Update rendering in index.html

Upcoming:
* Implement persistence to database.

### CODE REVIEW

File: app/main/routes.py [hunk 1]
```diff
@@ -1,22 +1,43 @@
...
+            generate_slang_form=generate_slang_form,
+            meaning_form=meaning_form,
+        )
+
+    return render_template(
+        "index.html", 
+        title="Home", 
+        slang_word="", 
+        generate_slang_form=generate_slang_form,
+        meaning_form=meaning_form)
```
Comment (Sasafrass) [inline app/main/routes.py:43]: @annaproxy is it possible to refactor the two title, slang_word, generate_slang_form and meaning_form parameters with **kwargs or smth?

Comment (annaproxy) [reply app/main/routes.py:43]: Yes, you can make a dictionary beforehand

`kwargs = {'title':'Home', 'meaning_form':meaning_form}`

Then call it twice
```
render_template(slang_word="Hell yes", **kwargs)
render_template(slang_word="", **kwargs)
```

Comment (annaproxy) [reply app/main/routes.py:43]: Unless the comment intention was "can you please do this" haha then I shall refactor it in a bit

Comment (Sasafrass) [reply app/main/routes.py]: Haha, added it in the new commit.

**Comment by annaproxy** [comment]
Is it easy to generate a word as soon as you save a word? To divide the number of user clicks by two

**Comment by Sasafrass** [comment]
> Is it easy to generate a word as soon as you save a word? To divide the number of user clicks by two
You mean generating a new word once you have saved your old word? If so, you can make an issue and I'll try to implement it later tonight.
> 
> Edit: the 'FIELD IS REQUIRED' scared me, as well, like 'what have I done wrong??'
This should be fixed in an upcoming commit. The field is technically (erroneously) always empty now because it's refreshed after I submit.

**Comment by Sasafrass** [comment]
Removed Work In Progress tag because it is a fully-fledged PR now.
Now includes the following implementations:
* Server-Side Session Management with Flask-Session (Also see https://github.com/Sasafrass/straattaal/pull/39 which was merged into this branch.
* Use session management to store the generated slang.
* Allow users to add their interpretation of the meaning of a word and persist both of this to the database.

**Review by annaproxy — APPROVED** [review]
LGTM

**Pull request merged**